sirepo/auth_role_moderation.py

```diff
@@ -22,11 +22,81 @@
 
 _cfg = None
 
+
+@sirepo.api_perm.require_adm
+def api_admModerate():
+
+    def _send_moderation_status_email(info):
+        approved_denied_subject = f'{info.app_name} Access Request '
+        email_subject = PKDict(
+            approve=approved_denied_subject + 'Approved',
+            clarify=f'Sirepo {info.app_name}: Additional Info?',
+            deny=approved_denied_subject + 'Denied',
+        )
+        if sirepo.auth.dev_and_no_method('email'):
+            pkdlog('Email not enabled. Moderation status is {}', info.status)
+            return
+        sirepo.smtp.send(
+            recipient=sirepo.auth.user_name(info.uid),
+            subject=email_subject.get(info.status),
```

Comment:
Using the constant:

```py
subject=_STATUS_TO_SUBJECT[info.status].format(info.app_name)
```

Reply:
ad61fc813610afe4d4c57cbe0e4322c503d6ac6c